OpenSubtitlesHandler/Util.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenSubtitlesHandler
+{
+    public static class Util
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
+        private static readonly JsonSerializerOptions SerializerOpts = new JsonSerializerOptions { IncludeFields = true, PropertyNamingPolicy = SnakeCaseNamingPolicy.Instance };
+        public static Action<string> OnHttpUpdate = _ => {};
```

Comment:
Where is this used?

Reply:
Hitting rate limits used to use this but it looks like it was lost somehow, I added it back in 837078e0342b4951cd25846fc26afaf4d2ded745

Reply:
Sorry for wait. I don't see a reason to use an Action, especially not if it's just for logging. I'd rather it was removed.

Reply:
Removed in a2b59f79206d579327a38f59a371bd4e86af363d